README.md

```diff
@@ -517,7 +525,15 @@ Check the [ImageOptim](https://github.com/toy/image_optim#configuration) to get
 | `optim`                           | ✔             |
 | `@optim`                          | ✗             |
 
-***By default `@optim` will use the default `jekyll`, otherwise you can provide `optim=preset` and have it used that preset.  ImageOptim provides advanced, and default as their default presets, you can define your own preset via Jekyll Assets configuration listed above.***
+***By default `@optim` will use the default configuration set `jekyll`, otherwise you can provide `optim=preset` and have it used that preset.  ImageOptim provides advanced, and default as their default presets, you can define your own preset via Jekyll Assets configuration listed above.***
+
+#### Example
+
+```liquid
+{% assets my_image.jpg @optim %}
+{% assets special.jpg optim=specialPreset %}
+```
+
```

Comment:
I feel this defeats the purpose of the arguments list, in it's entirety.  I'm hesitant to accept this change, because all things considered, it provides no additional context, but it also does no harm, so I guess I'm willing to accept it if you adjust it to use the standard "asset.jpg" or "img.jpg", not the names you currently have.

As well, in Ruby we don't do "specialPreset" we would do "special_preset"

Reply:
While I get it, the documentation is really hard to read, and the arguments list didn't exactly struck me as part of the "Image Optim"  documentation.

Maybe splitting the doc in a few separated files instead, i find it quite hard to read this way, examples and files names, and others contexts would be of much help for newcomers (my advice at least).